syntax_and_semantics/if_var.md

```diff
@@ -35,7 +35,7 @@ Of course, reassigning a variable inside the `then` branch makes that variable h
 
 ## Limitations
 
-The above logic **doesn’t** work with instance variables, class variables and variables bound in a closure. The value of these kinds of variables could potentially be affected by another fiber after the condition was checked, rendering it `nil`.
+The above logic works **only for local variables**. It doesn’t work with instance variables, class variables, variables bound in a closure and constants. The value of these kinds of variables could potentially be affected by another fiber after the condition was checked, rendering it `nil`.
```

Comment:
> The value of these kinds of variables could potentially be affected by another fiber after the condition was checked

This is wrong for constants though.. since they can't be changed

Reply:
> The value of these kinds of **variables**

Reply:
yeeah right ok ^^